helpers/fuzzybool.go

```diff
@@ -0,0 +1,66 @@
+package helpers
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	// FuzzyBool resolves boolean sent as strings, integers, float64 or boolean, json unmarshal defaults to true
+	FuzzyBool bool
+)
+
+// UnmarshalJSON FuzzyBool
+func (sb *FuzzyBool) UnmarshalJSON(arg []byte) error {
+	if sb == nil {
+		return nil
```

Comment:
Is there some reason I am missing for not throwing an error here? Better fail fast.

Reply:
is it an error? I see it as "nothing to unmarshal" ?

Reply:
I think of it like there is no place to put the unmarshalled raw json.

Reply:
It is a minor, so if we disagree it can stay as is